cli/src/commands/squid/release.ts

```diff
@@ -49,6 +69,22 @@ export default class Release extends Command {
             nameAndVersion,
             this
         );
+
+        const envs: Record<string, string> = {} 
+        
+        flags.env?.forEach((e: string)=>{
+            const v = getEnv(e);
+            envs[v.name] = v.value;
+        });
+
+        if (flags.envFile != undefined && existsSync(flags.envFile)) {
```

Comment:
[DRY
](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)

Make a function to read file and parse

Reply:
probably done not the best variant but safe